ion-hash/tests/ion_hash_tests.rs

```diff
@@ -150,8 +150,8 @@ fn test_case<E: Element>(ion: &E, strukt: &E) -> IonResult<()> {
     // variable sized result. Without this, any test failure will write lots of
     // stuff to your console which can be annoying since it takes forever.
     assert_eq!(
-        without_trailing_zeros(&expected[..]),
-        without_trailing_zeros(&result[..]),
+        format!("{:02x?}", without_trailing_zeros(&expected[..])),
+        format!("{:02x?}", without_trailing_zeros(&result[..])),
```

Comment:
This change makes debugging easier because the failures look like the hex in the test files.

Reply:
Also there is [`pretty_hex`](https://docs.rs/pretty-hex/0.2.1/pretty_hex/index.html) which I use in the element round tripping tests IIRC, which is handy (no need to change this, just an FYI).